### DISCUSSION TITLE
@Holy shit!!! Guard’s Inexcusable Disruption: Immediate Rectification Demanded NOW

### DISCUSSION THREAD

**Discussion by jameschung0225** [List-KR/community#22] [General]
AdGuard's disruptive actions have reached a point of inexcusability. Your service has caused a full-scale shutdown of an essential banking application in Korea, by blocking the DNS resolution for the domain mbp.hanabank.com. 

This egregious disregard for service reliability is entirely insupportable and goes beyond simple inconvenience; it's a matter of financial stability for numerous users. I insist upon an immediate, unequivocal rectification of this issue. 

Consider this as a stern, formal demand for IMMEDIATE action to restore the DNS resolution for mbp.hanabank.com and return the banking application to full operation. Any further delay will be considered as a blatant refusal of your responsibilities. 

I expect an immediate response and swift resolution of the issue. Consider this a final warning.

**Comment by piquark6046** [comment]
Duplicate of List-KR/community#17
